src/main/java/duke/commands/EventCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.commands;
+
+import duke.tasks.Event;
+import duke.outputs.Messages;
+import duke.file.TaskList;
+import duke.ui.*;
+
+/**
+ * Command when adding a new event task to tasklist
+ */
+public class EventCommand extends Command {
+    private Event event;
+
+    /**
+     * creates a new event task
```

Comment:
Do check the capitalisation and use of periods in your comments are in line with the coding standards. 

https://se-education.org/guides/conventions/java/basic.html#comments